MatrixSDK/Crypto/MXCrypto.h

```diff
@@ -102,48 +102,11 @@ extern NSString *const MXDeviceListDidUpdateUsersDevicesNotification;
  */
 @property (nonatomic, readonly) MXRecoveryService *recoveryService;
 
-/**
- The secret storage on homeserver manager.
- */
-@property (nonatomic, readonly) MXSecretStorage *secretStorage;
```

Comment:
All of these (apart from deprecated `decryptEvent` and duplciate `trustLevelSummary`) are just moved to `MXLegacyCrypto` without change